win/stdatomic.c

```diff
@@ -0,0 +1,16 @@
+#include <stdio.h>
+#include <intrin.h>
+
+#include "stdatomic.h"
+
+int atomic_compare_exchange_strong(atomic_int *ptr, int *expected, int desired)
+{
+	int comparand = *expected;
+	int dstValue = _InterlockedCompareExchange((volatile long*)ptr, desired, comparand);
```

Comment:
i don't understand well when someone should use _InterlockedCompareExchange or _InterlockedCompareExchange8 16 64 :joy:

Reply:
I just reused this function from another project.

Reply:
Currently I'm not able to compile project at all.